04/app.js

```diff
@@ -3,6 +3,25 @@ document.addEventListener('DOMContentLoaded', init);
 function init() {
     const boxElement = document.querySelector('.box');
     setBoxShadow(boxElement, '#000000');
+
+    const colorInput = boxElement.previousElementSibling.children[0];
+    const rangeInput = boxElement.previousElementSibling.children[1];
+
+    colorInput.addEventListener('change', function (e) {
+        const colorValue = e.target.value;
+        setBoxShadow(boxElement, colorValue)
```

Comment:
Przydałoby się podawać aktualną wartość dla "opacity", a nie domyślną bo przy zmianie koloru (jeśli opacity wynosi mniej niż 1) to mamy lekkie zakłamanie)